subprojects/xsts/xsts-analysis/src/test/java/hu/bme/mit/theta/xsts/analysis/XstsTest.java

```diff
@@ -131,14 +131,16 @@ public static Collection<Object[]> data() {
                 {"src/test/resources/model/choices.xsts", "src/test/resources/property/choices.prop",
                         false, XstsConfigBuilder.Domain.EXPL_PRED_COMBINED},
 
-                {"src/test/resources/model/literals.xsts", "src/test/resources/property/literals.prop",
```

Comment:
Please add 1-2 tests instead of this removed test: one checking that literals of the same type can't be equal, and one checking that literals of different types with the same name can't be equal.

Reply:
Done in e757a60